mkosi/__init__.py

```diff
@@ -3222,7 +3284,7 @@ def run_postinst_script(
             nspawn_params += nspawn_params_for_blockdev_access(args, loopdev)
 
         run_workspace_command(args, root, ["/root/postinst", verb],
-                              network=(args.with_network is True),
+                              network=(args.with_network is True and args.use_mirror_verbatim is False),
```

Comment:
```suggestion
                              network=(args.with_network is True and not args.use_mirror_verbatim),
```
The `args.with_network is True` check is because it can also have the string value `never`